rosco-web/config/rosco.yml

```diff
@@ -255,11 +255,11 @@ openstack:
       virtualizationSettings:
       - region: r1
         instanceType: smem-2vcpu
-        sourceImageName: ubuntu-latest
+        sourceImageId: b12f2467-cfca-4a65-b29b-d11b8c46818d
```

Comment:
Would it make sense to drop a link here so a reader knows where this value is retrieved from? And does this have to be kept in sync somehow?